src/errors.ts

```diff
@@ -33,8 +33,16 @@ class WebhookAuthenticateError extends Error {
   }
 }
 
+class AttachmentDecryptionError extends Error {
+  constructor(message = 'Attachment decryption with the given nonce failed.') {
+    super(message)
+    this.name = this.constructor.name
```

Comment:
it isn't! i actually left it in because i thought it was convention @__@ i'll remove it!

Reply:
i went to investigate more and it was q enlightening for me! the reason why this is done is because it's an error. when js (and by extension, ts) throws an error, the error printed to the output is done using  `error.name + ' ' + error.message` - this means that when the error here is thrown, because it's missing the name property, it will default to `Error` (as it calls `super(message)`). While the argument might be that it's guaranteed to never be thrown, i think this is a relatively small change that helps in the future so i'd just leave it in!